apiary.apib

```diff
@@ -10,63 +10,63 @@ Customer API is a simple microservice allowing consumers to view polls and vote
 
 Attributes
 - `id` - Customer's ID
-- `created_at` - UTC timestamp ([ISO 8601](http://www.w3.org/TR/NOTE-datetime)) for when this object was created
-- `updated_at` - UTC timestamp ([ISO 8601](http://www.w3.org/TR/NOTE-datetime)) for last time this object was updated - *optional*
-- `group_id` - Customer's group (for Hunt Club members, groupe_id == 4)
+- `createdAt` - UTC timestamp ([ISO 8601](http://www.w3.org/TR/NOTE-datetime)) for when this object was created
+- `updatedAt` - UTC timestamp ([ISO 8601](http://www.w3.org/TR/NOTE-datetime)) for last time this object was updated
+- `groups` - Customer's group as array of strings (Hunt Club members, ["huntclub"])
```

Comment:
You're right. The point here is that we're going out of Magento. My gut feeling tells me we need a broader definition (think of segmentation in our esp).